readme.md

```diff
@@ -27,7 +27,7 @@ var remark = require('remark')
 var toString = require('mdast-util-to-string')
 var Index = require('unist-util-index')
 
-// Read this readme:
+// parse the readme.md of this repo:
```

Comment:
```suggestion
// Parse and read this repo’s readme:
```

How about this? It still includes that the file is also read.

Reply:
great :+1: :)

Reply:
updated the PR